### PR TITLE
Added Sample references for Vision Catalog

### PR DESCRIPTION
**Pull request by harshithapv**
Added Sample references for Vision Catalog

### CODE REVIEW

File: dotnet/xml/Microsoft.ML/VisionCatalog.xml [hunk 1]
```diff
@@ -79,6 +79,14 @@
         </summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
+        <example>
+          <format type="text/markdown">
+            <![CDATA[
+             [!code-csharp[ImageClassification](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs)]
```
Comment (natke) [inline dotnet/xml/Microsoft.ML/VisionCatalog.xml]: Would we want the default sample to come first?

Comment (natke) [reply dotnet/xml/Microsoft.ML/VisionCatalog.xml]: @codemzs ?

Comment (codemzs) [reply dotnet/xml/Microsoft.ML/VisionCatalog.xml]: Probably. 

---
In reply to: [343390917](https://github.com/dotnet/ml-api-docs/pull/116#discussion_r343390917) [](ancestors = 343390917)

**Comment by codemzs** [comment]
@natke All catalogs are part of Microsoft.ML namespace which every user will use and the purpose of catalog is to discover APIs. Once you try to use ImageClassification API intellisense will suggest you reference Microsoft.ML.Vision nuget that contains ImageClassificationTrainer.

File: dotnet/xml/Microsoft.ML/VisionCatalog.xml (hunk 1 above)
Comment (natke) [reply dotnet/xml/Microsoft.ML/VisionCatalog.xml]: Looks good. It should be a cref though. Have a look at: https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.standardtrainerscatalog?view=ml-dotnet as an example

**Pull request merged**